fieldservice_change_management/report/change_log_reports.xml

```diff
@@ -0,0 +1,93 @@
+<odoo>
+  <report id="change_log_document"
+          model="change.log"
+          string="Change Logs"
+          name="fieldservice_change_management.report_change_log_document"
+          file="fieldservice_change_management.report_change_log_document"
+          report_type="qweb-pdf"/>
+
+  <template id="report_change_log_document">
+    <t t-name="fieldservice_change_management.report_change_log_document">
+      <t t-name="fieldservice_change_management.report_change_log_document">
+        <t t-call="web.html_container">
+          <t t-call="web.external_layout">
+            <div align="center">
+              <h3>Change Logs</h3>
+            </div>
+            <t t-foreach="docs" t-as="doc">
+              <h4><span t-field="doc.location_id.ref"/> - <span t-field="doc.location_id"/></h4>
+              <table class="table table-sm o_main_table">
+                <thead>
+                    <tr>
+                        <t t-set="colspan" t-value="4"/>
+                        <th class="text-left">Title</th>
+                        <th class="text-left">Type</th>
+                        <th class="text-left">Description</th>
+                        <th class="text-left">Changed On</th>
+                        <th class="text-left">Changed By</th>
+                    </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td><span t-field="doc.name"/></td>
+                    <td class="text-left"><span t-field="doc.type_id"/></td>
+                    <td class="text-left"><span t-field="doc.description"/></td>
+                    <td class="text-left"><span t-field="doc.change_date"/></td>
```

Comment:
KeyError: 'change_date'

Error to render compiling AST
KeyError: 'change_date'
Template: 1449
Path: /templates/t/t/t/t/t/t/table/tbody/tr/td[4]/span
Node: <span t-field="doc.change_date"/>

```suggestion
                    <td class="text-left"><span t-field="doc.implemented_on"/></td>
```